packages/web/components/Player/PlayerProfilePicture.tsx

```diff
@@ -8,11 +9,13 @@ import { getPlayerImage } from 'utils/playerHelpers';
 
 type PlayerProfilePictureProps = AvatarProps & {
   player?: Player | GuildPlayer;
+  round?: boolean;
```

Comment:
This should be its own component `PlayerProfilePictureRound` since the display is entirely different based on the prop

Reply:
moved into it's own component